src/pybind/mgr/dashboard_v2/controllers/summary.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+
+import cherrypy
+
+from ..tools import AuthRequired, ApiController, BaseController, ViewCache
+from ..services.ceph_service import CephService
+
+
+@ApiController('summary')
+class Summary(BaseController):
+    @ViewCache()
+    def _rbd_pool_ls(self):
```

Comment:
This function does not need to be decorated with `@ViewCache` because the implementation of getting the list of pools just consults the `osd_map` that is stored in the ceph-mgr memory, no connection is done to the cluster,